README.md

```diff
@@ -11,12 +11,13 @@ of infrastructure providers and types.  In principle, any of them can be program
 
 If you want to wrap a _new_ Terraform provider as a Pulumi provider, check out [pulumi/pulumi-tf-provider-boilerplate](https://github.com/pulumi/pulumi-tf-provider-boilerplate).
 
-## Overview
+## Developing a New Provider
+
+The recommended way to start developing a new TF provider is with https://github.com/pulumi/pulumi-tf-provider-boilerplate/issues.
```

Comment:
Boilerplate link is appreciated, this is much more practical than from scratch instructions. Nit: should it link to /issues?

How shall we approach sdkv2, muxed, pf providers? Parametric boilerplate? Several boilerplates? Code generators? 

Worth dusting off either @tmeckel cookie-cutter or @AaronFriel's tool as a starting point for making a provider instead of the boilerplate?

Reply:
> Nit: should it link to /issues?

Fixed.

> How shall we approach sdkv2, muxed, pf providers? Parametric boilerplate? Several boilerplates? Code generators?
>
>Worth dusting off either @tmeckel cookie-cutter or @AaronFriel's tool as a starting point for making a provider instead of the boilerplate?

Yes. I'm not a big fan of boilerplates, so my intuition is to adapt a tool, @tmeckel's or @AaronFriel's both seem like good starting points to investigate.